proxy_error.html

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><methodResponse><params><param><value><struct><member><name>resultCode</name><value><int>998</int></value></member></struct></value></param></params></methodResponse>
```

Comment:
Would you think a readable layout should be considered bad?

```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<methodResponse>
    <params>
        <param>
            <value>
                <struct>
                    <member>
                        <name>resultCode</name>
                        <value>
                            <int>998</int>
                        </value>
                    </member>
                </struct>
            </value>
        </param>
    </params>
</methodResponse>
```